pkg/sosreport/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "version": 0,
+
+    "tools": {
+        "index": {
+            "label": "Create diagnostic report"
```

Comment:
This needs to be marked as translatable somewhere. Currently I usually just use the same thing as the <title> and mark it as translatable.